PlateSpace/ETL/index.js

```diff
@@ -0,0 +1,78 @@
+var axios = require("axios");
+var flatten = require("lodash.flatten");
+var mongoose = require("mongoose");
+var config = require("./config");
+
+axios.defaults.headers.common["Authorization"] = config.YELP_AUTH_TOKEN;
+
+var i = 0;
+const openingHours = [
+  { start: "0800", end: "2200" },
+  { start: "0900", end: "2300" },
+  { start: "1000", end: "2400" }
+];
+
+function createPlateSpace(data) {
```

Comment:
since you're calling this function from `.map()` (L66 below) you can let this function take a second arg, which is the index. No need to maintain a separate global counter, which is also a bit of a code smell.